### PR TITLE
Improvement of API requests error handling

### PR DESCRIPTION
**Pull request by sneridagh**
https://github.com/plone/plone.rest/issues/101 is solved now, however, on GET we can't get a 403 now :( https://github.com/plone/plone.restapi/issues/893

@robgietema do you want to take a look?

/cc @tisto

### CODE REVIEW

**Comment by tisto** [comment]
@sneridagh w00t! Awesome!!!

**Comment by tisto** [comment]
Seems a Cypress test (folder contents) fails on both GH actions and Travis.

File: src/middleware/api.js [hunk 1]
```diff
@@ -8,7 +8,13 @@ import jwtDecode from 'jwt-decode';
...
 import { settings } from '~/config';
 
-import { LOGIN } from '@plone/volto/constants/ActionTypes';
+import {
+  LOGIN,
+  RESET_APIERROR,
+  SET_APIERROR,
+} from '@plone/volto/constants/ActionTypes';
+
+const ACTIONS_RAISING_ERRORS = ['GET_CONTENT', 'UPDATE_CONTENT'];
```
Comment (tiberiuichim) [inline src/middleware/api.js:17]: Could this be read from config.settings? This way it would be extensible without customizing api.js

**Comment by sneridagh** [comment]
@tiberiuichim done as suggested! I think now will be green

**Pull request merged**